source/retryable-writes/retryable-writes.rst

```diff
@@ -52,65 +51,57 @@ Transaction ID
    The transaction ID identifies the transaction as part of which the command is
    running. In a write command where the client has requested retryable
    behavior, it is expressed by the top-level ``sessionId`` and ``txnNum``
-   fields. The ``sessionId`` component is the corresponding server session ID,
-   which is an opaque BSON value returned by the ``startSession`` command. The
-   ``txnNum`` component is a monotonically increasing (per server session),
+   fields. The ``sessionId`` component is the corresponding server session ID.
+   which is an opaque BSON value defined in the Driver Session specification.
+   The ``txnNum`` component is a monotonically increasing (per server session),
    positive 64-bit integer.
 
 DriverSession
-   Driver object representing a client session and the operations that can be
-   performed on it, as defined in the Driver Session specification. This object
-   is always associated with a server session. The name of this object MAY vary
-   across drivers.
-
-SessionOptions
-   Data structure defined in the Driver Session specification to provide options
-   when creating a DriverSession. This could be a strongly typed class in some
-   drivers, or it could be a loosely typed dictionary in other drivers.
+   Driver object representing a client session, which is defined in the Driver
+   Session specification. This object is always associated with a server
+   session; however, drivers will pool server sessions so that creating a
+   DriverSession will not always entail creation of a new server session. The
+   name of this object MAY vary across drivers.
```

Comment:
Removed SessionOptions entirely and referenced Driver Session spec re: creation of DriverSession objects. The bit about pooling is relevant to our use case below where we might create these internally for a single operation.